### DISCUSSION TITLE
Characters with accents

### DISCUSSION THREAD

**Discussion by HerediaSM** [Bodmer/TFT_eSPI#2964] [Q&A - Fonts] [answered]
Dear Bodmer,
How can I use special characters (accents) while retaining the default font provided by this library?
ESP32 - TFT ST7735R

If this is not possible, what do you recommend me to use? 

Thanks in advance.

**Comment by Bodmer** [comment, marked as answer]
The built in fonts were originally created to run on an Uno and Mega so compactness rather than exensibility was in mind.

There are a few options for accented characters:
1. Smooth (antialiased) fonts - created with the provided [Processing IDE sketch](https://github.com/Bodmer/TFT_eSPI/blob/master/Tools/Create_Smooth_Font/Create_font/Create_font.pde)
2. Bitmap fonts using [U8g2_for_TFT_eSPI](https://github.com/Bodmer/U8g2_for_TFT_eSPI)
3. TrueType fonts using library [here](https://github.com/takkaO/OpenFontRender).

**Reply by HerediaSM** [reply]
The third option was useful to me. Thank you.
